mix.exs

```diff
@@ -19,6 +20,12 @@ defmodule ExDoc.Mixfile do
     [clean: [&clean_test_fixtures/1, "clean"]]
   end
 
+  defp package do
+   [licenses: ["Apache 2.0"],
+    links: %{"GitHub" => "https://github.com/elixir-lang/ex_doc",
+             "Contributors" => "https://github.com/elixir-lang/ex_doc/graphs/contributors"}]
```

Comment:
Yes, normally it's a list of names, but in this case it's a long list of contributors, that's why I added a link to the contributors page instead.

I already push and update, but, feel free to close this PR if you don't like it!

Just for reference, It's easy to retrieve the names of the contributors:

```bash
$ curl https://api.github.com/repos/elixir-lang/ex_doc/contributors | grep "login" | awk -F: '{print $2}' | perl -lne 's/[\s\",]//g; print' > users
$ for u in $(cat users); do curl https://api.github.com/users/$u | grep "name" | awk -F: '{print $2}' >> contributors; done
```